talker_agent/Dockerfile

```diff
@@ -8,9 +8,18 @@ RUN mkdir -p /root/talker
 WORKDIR /root/talker
 
 RUN echo {\"password\": null, \"port\": 6379, \"host\": \"redis\", \"host_id\": \"$TALKER_HOST_ID\"} > conf.json
-COPY talker.py install_agent.sh ./
+COPY talker.py talker-service install_agent.sh ./
```

Comment:
sort these steps in this order:
1. os config (installing dependencies, locale etc)
2. talker service copy and setup
3. talker.py copy and setup